### ISSUE TITLE
Question on robosuite eval-hard setting

### ISSUE THREAD

**Issue by SMLee0407** [gemcollector/RL-ViGen#19]
Hello.
As shown in envs/robosuiteVGB/robosuitevgb/vgb_wrapper.py, line 119: assert mode in ["train", "eval-easy", "eval-medium", "eval-hard"],
I changed mode as "eval-hard" to test on eval-hard setting, but video_background was not applied.
In /envs/robosuiteVGB/robosuitevgb/utils.py, line 63,  
elif cfg_dict['mode'] == 'eval_hard':   
-> elif cfg_dict['mode'] == 'eval-hard':    
works. 

Also, in "medium" setting,  cfg_dict['except_robot'] = False. But, in "hard" setting, cfg_dict['except_robot'] = True.
I wonder why "cfg_dict['except_robot'] = True" in "hard" setting although "medium" setting is False for the option.
(It is easier when cfg_dict['except_robot'] = True.)

Thanks.

**Comment by gemcollector** [comment]
Hello, there! Thanks for your careful checking! I have fixed this bug. 
I tried the `eval-hard` setting, I can acquire the video background as follows:
![change-video-background](https://github.com/user-attachments/assets/9047368d-1163-4ffb-af30-2c7aa276bfca)

The reason is that the `eval-hard` includes the dynamic changing background, which is different from the static `eval-medium` level. I found that if you change the robot texture and include the dynamic video background together, all agents will fail the task. It is too challenging for the existing method. Therefore, I encourage the researcher to  explore a novel method to handle this kind of scenario.
